services/elasticache/pkg/resource/snapshot/custom_set_output.go

```diff
@@ -0,0 +1,89 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package snapshot
+
+import (
+	ackv1alpha1 "github.com/aws/aws-controllers-k8s/apis/core/v1alpha1"
+	svcapitypes "github.com/aws/aws-controllers-k8s/services/elasticache/apis/v1alpha1"
+	"github.com/aws/aws-sdk-go/service/elasticache"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func (rm *resourceManager) CustomDescribeSnapshotSetOutput(
+	r *resource,
+	resp *elasticache.DescribeSnapshotsOutput,
+	ko *svcapitypes.Snapshot,
+) *svcapitypes.Snapshot {
+	if len(resp.Snapshots) == 0 {
+		return ko
+	}
+	elem := resp.Snapshots[0]
+	rm.customSetOutput(r, elem, ko)
+	return ko
+}
+
+func (rm *resourceManager) CustomCreateSnapshotSetOutput(
+	r *resource,
+	resp *elasticache.CreateSnapshotOutput,
+	ko *svcapitypes.Snapshot,
+) *svcapitypes.Snapshot {
+	rm.customSetOutput(r, resp.Snapshot, ko)
+	return ko
```

Comment:
Here, why don't we set `Spec.ReplicationGroupID`?

Reply:
Thanks! updated to set `Spec.ReplicationGroupID`